commit bc776706738359ac15c726b150a9267285210367
Merge: 3d7d034 55ef04a
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Tue May 31 10:29:37 2016 -0700

    Merge pull request #3929 from thinkyhead/rc_fix_printrboard_dac
    
    Need to call dac_init() in setup() for DAC_STEPPER_CURRENT

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index b3c302d..88a25fe 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -856,6 +856,10 @@ void setup() {
     digipot_i2c_init();
   #endif
 
+  #if ENABLED(DAC_STEPPER_CURRENT)
+    dac_init();
+  #endif
+
   #if ENABLED(Z_PROBE_SLED)
     pinMode(SLED_PIN, OUTPUT);
     digitalWrite(SLED_PIN, LOW); // turn it off
